package.json

```diff
@@ -37,6 +37,7 @@
     "@zendeskgarden/react-avatars": "8.19.0",
     "@zendeskgarden/react-breadcrumbs": "8.19.0",
     "@zendeskgarden/react-buttons": "8.19.0",
+    "@zendeskgarden/react-chrome": "8.19.0",
```

Comment:
Can this be set to `8.21.1` so that when it's merged with #165, all things are same-same?